### PR TITLE
Update README.md - Runtimes on Fellowship Repo

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

**Review by liamaharon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ way. The Polkadot SDK comprises three main pieces of software:
 [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](./polkadot/LICENSE)
 
 Implementation of a node for the https://polkadot.network in Rust, using the Substrate framework. This directory
-currently contains runtimes for the Polkadot, Kusama, Westend, and Rococo networks. In the future, these will be
-relocated to the [`runtimes`](https://github.com/polkadot-fellows/runtimes/) repository.
+currently contains runtimes for the Westend and Rococo test networks. Polkadot and Kusama runtimes are located in the [`runtimes`](https://github.com/polkadot-fellows/runtimes/) repository maintained by
```
Comment (ggwpez) [inline README.md:21]: Maybe also mention their system parachains?

Comment (DrW3RK) [reply README.md]: Done

**Review by bkchr — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ way. The Polkadot SDK comprises three main pieces of software:
 [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](./polkadot/LICENSE)
 
 Implementation of a node for the https://polkadot.network in Rust, using the Substrate framework. This directory
-currently contains runtimes for the Polkadot, Kusama, Westend, and Rococo networks. In the future, these will be
-relocated to the [`runtimes`](https://github.com/polkadot-fellows/runtimes/) repository.
+currently contains runtimes for the Westend and Rococo test networks. Polkadot, Kusama and their system chain runtimes
```
Comment (bkchr) [inline README.md:21]:
```suggestion
currently contains runtimes for the Westend and Rococo test networks. Polkadot, Kusama and their system chain runtimes
```

**Review by ggwpez — APPROVED** [review]
🙏

**Pull request merged**